### DISCUSSION TITLE
Product Metrics count inconsistency

### DISCUSSION THREAD

**Discussion by jugalb-icpl** [DefectDojo/django-DefectDojo#11452] [General]
Hi everyone,

I’ve noticed a discrepancy in the product count metrics and need some clarification.

Observation 1:
In the Product Metrics page, the metric for Open Findings (All Time) for the product "Bodgelt" shows a total of 339 findings (see the first image below).

![image](https://github.com/user-attachments/assets/3be629b3-603f-4c85-91c3-5384bb02b50d)

Observation 2:
When I click on the product's hyperlink to view the findings, the resulting page displays only 205 total findings (see the second image below).

![Screenshot 2024-12-21 144734](https://github.com/user-attachments/assets/c726c571-e77d-41cf-bf20-5df56489fc56)

Observation 3:
After some experimentation, I found that the count displayed under Open Findings(date=any) in Product Metrics appears to represent all findings filtered by the "Verified" status (see the third image and its URL).

![Screenshot 2024-12-21 144743](https://github.com/user-attachments/assets/5cf0e36a-8ff9-47e1-99f4-462bdf396e4e)

Question:
Am I misunderstanding the data or the purpose of these metrics? Or is this an issue with how the counts are being calculated/displayed?

Looking forward to your insights! Thanks in advance.

**Comment by valentijnscholten** [comment]
The first screenshot looks different from what I see in my instance, it shows the period with a start date and end date.

But the 'Open**ed** findings' metrics shows the number of findings that were opened in this period. Even if they are now closed, they are still being counted towards the metric because they were created/opened in the selected period.. Whereas your second screenshot only the open findings are shown. Does that make sense?
